scripts/fast-reboot

```diff
@@ -15,10 +17,48 @@ then
     exit 1
 fi
 
+function debug()
+{
+    if [[ x"${VERBOSE}" == x"yes" ]]; then
+        echo `date` $@
```

Comment:
>echo [](start = 8, length = 4)

This is a script for shutdown. Left message on stdout may be the best choice. How about syslog?

Reply:
Good idea. I also send these debug output to syslog.

I think you meant to say not leaving messages on stdout. But I think that is what verbose supposed to mean.